src/policy/copyspace.rs

```diff
@@ -215,6 +217,7 @@ impl<VM: VMBinding> CopySpace<VM> {
             new_object
         } else {
             trace!("... no it isn't. Copying");
+            let copy_context = unsafe { worker.local::<CopySpaceCopyContext<VM>>() };
             let new_object =
                 object_forwarding::forward_object::<VM, _>(object, semantics, copy_context);
```

Comment:
`copy_context` contains information about where the object is copied **to**.  For example, if this CopySpace is used as the nursery of the GenImmix plan, then the `copy_context` here should be `ImmixCopyContext`, not `CopySpaceCopyContext`.

Reply:
Yeah. Exactly. I am working on this. This PR is not yet ready for review.